### ISSUE TITLE
CONFIGURE: r-cran-devtools not available

### ISSUE THREAD

**Issue by gjwgit** [mlhubber/mlhub#88]
On a fresh DSVM mlhub tries to install r-cran-devtools which is not available.

```
$ ml configure
...
*** Installing the system package 'r-cran-devtools' ...
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package r-cran-devtools is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
...
An error was encountered:

E: Package 'r-cran-devtools' has no installation candidate
Error in contrib.url(repos, type) : 
  trying to use CRAN without setting a mirror
Calls: install.packages -> grep -> contrib.url
Execution halted

```

Workaround:

`$ Rscript -e 'install.packages("devtools", repos="https://cran.r-project.org")'`

**Comment by simonzhaoms** [comment]
This scenario was considered:

https://github.com/mlhubber/mlhub/blob/2defed036cf88e551ff7c146e77baa7ccbbaa3b4/mlhub/scripts/dep/mlhub.sh#L37-L62

I am not sure why the code doesn't work.  I need to figure it out.

**Comment by simonzhaoms** [comment]
Fixed in 9390af5
